src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -433,6 +427,15 @@ public static void configure(final File file, final String defaultIdent) {
     StatusPrinter.printInCaseOfErrorsOrWarnings(context);
   }
 
+  private static LoggerContext setupLoggerContext(Logger rootLogger, String ident) {
+    final LoggerContext context = rootLogger.getLoggerContext();
+    context.reset();
+    context.putProperty("ident", ident);
+    context.putProperty("pid", getMyPid());
+    context.putProperty("hostname", getHeliosHostname());
```

Comment:
the hostname property is used at https://github.com/spotify/logging-java/pull/14/files#diff-a65ed27cbf34f0b2a250058370265fd0R56. if it is not set (or null) then the default value is used.

i'll update this code to not set the hostname if the environment variable is also not set